styles.css

```diff
@@ -1,50 +1,89 @@
 
 
-
-
 body {
   background-color: #c1cdcd;
   margin: 0;
   padding: 0;
-  font-family: Georgia, 'Times New Roman', Times, serif;
   display: grid;
-  grid-template-columns: 1fr 3fr;
+  grid-template-columns: 1fr 4fr;
   grid-template-areas: "sidebar content";
+  font-family: Caveat, Georgia, 'Times New Roman', Times, serif;
+  font-size: 1.1em;
 }
 
 nav {
-  background-color: #c1ffc1;
+  background-color: #ffefdb;
   grid-area: sidebar;
   display: grid;
   grid-template-columns: 1fr;
   grid-template-rows: 40px auto;
   min-height: 100vh;
+  font-family: "Grenze Gotisch", Times, serif;
 }
 
 nav>header {
```

Comment:
I guess the formatter page might be doing this, but leave spaces around the `>`

Reply:
Yeah, that was the beautifyer. Fixing now.